### PR TITLE
Add a kano-content sync before loading the profile

### PR DESCRIPTION
**Pull request by convolu**

### CODE REVIEW

**Comment by convolu** [comment]
@alex5imon @pazdera @tombettany Could you please review?

**Comment by convolu** [comment]
Bump

**Comment by pazdera** [comment]
Looks good! Damn sudo :-).

**Comment by alex5imon** [comment]
@convolu do we want to add some sort of dialog to notify users that we are syncing?

File: bin/kano-profile-gui [hunk 1]
```diff
@@ -164,6 +167,9 @@ class MainWindow(ApplicationWindow):
 if __name__ == '__main__':
     args = docopt.docopt(__doc__)
 
+    if is_internet():
+        run_cmd('sudo kano-content sync')
```
Comment (tombettany) [inline bin/kano-profile-gui]: Is there not a python function in kano-content which can do the sync, rather than having to launch it as a subprocess?

Comment (convolu) [inline bin/kano-profile-gui]: There is one but unfortunately, it requires elevated privileges to run

**Comment by convolu** [comment]
@alex5imon It shouldn't be taking too long, even from a fresh kit. Most of the cases it exits right away (because there is nothing new to sync).

File: bin/kano-profile-gui (hunk 1 above)
Comment (tombettany) [inline bin/kano-profile-gui]: Fair enough. In which case this is fine with me.

**Comment by convolu** [comment]
Closing this PR, see https://github.com/KanoComputing/peldins/issues/1962 for details

**Pull request closed without merging**